README.md

```diff
@@ -98,15 +87,7 @@ correcting any deficencies that may exist. This is easily done like so:
 
 Now, with the "bootstrap" VM properly configured, we can run the build:
 
-    $ ./scripts/docker-run.sh gradle ...
-
-This will create a new container based on the image we previously
-created, and then execute "gradle" inside of that container.
-
-The "./scripts/docker-run" script can also be run without any arguments,
-which will provide an interactive shell running in the container
-environment, with the appliance-build git repository mounted inside of
-the container; this can be useful for debugging and/or experimenting.
+    $ sudo -E ./gradlew ...
```

Comment:
What environment variables do we need to preserve here?

Reply:
In general (e.g. when building manually), none are required. But it's possible to specify variables that cause the build to do different things; which is functionality used by Jenkins to pass in different values.

I could remove the suggestion of using `-E` here, since those reading this doc likely won't be specifying any custom values.

Should I remove it from here?

Reply:
I think it makes sense to remove it. It makes the instructions a little simpler, and if people do want to do more advanced things, they can always `sudo WHATEVER=whatever ./gradlew ...`